packages/ckeditor5-table/tests/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -1291,46 +1292,137 @@ describe( 'TableColumnResizeEditing', () => {
 					}
 				} );
 
-				it( 'shrinks the table twice as much when resizing centered table as compared to aligned table', () => {
-					const columnToResizeIndex = 1;
-					const mouseMovementVector = { x: -10, y: 0 };
+				describe( 'where there are 2 tables: centered and aligned', () => {
```

Comment:
```suggestion
				describe( 'with 2 tables: centered and aligned', () => {
```

Reply:
I know that it's from the code that was before but this wording is a little unfortunate.